src/app/error-handler/UncaughtExceptionHandler.service.ts

```diff
@@ -5,11 +5,9 @@ import {ApiSettings} from '../api-connector/api-settings.service';
 
 // ---Before Send Function
 // Runs before the Ajax Appender sends the logs to the server
-function beforeSendFunction(xhr: XMLHttpRequest, json: JSON): any {
-  json = filter(json); // filter out duplicate error messages
+function beforeSendFunction(xhr: XMLHttpRequest): any {
```

Comment:
json argument and filter function are defined and used, please do not delete.
see https://js.jsnlog.com/Documentation/HowTo/AddingRequestHeaders for more information

Reply:
resolved